Source/WebCore/dom/TrustedType.cpp

```diff
@@ -289,25 +289,27 @@ ExceptionOr<String> requireTrustedTypesForPreNavigationCheckPasses(ScriptExecuti
     auto decodedURLString = PAL::decodeURLEscapeSequences(urlString);
     auto scriptSource = decodedURLString.substring(javascriptSchemeLength);
 
+    VM& vm = scriptExecutionContext.vm();
+    auto throwScope = DECLARE_THROW_SCOPE(vm);
+
     auto convertedScriptSource = processValueWithDefaultPolicy(scriptExecutionContext, expectedType, scriptSource, sink);
-    if (std::holds_alternative<Exception>(convertedScriptSource))
-        return WTFMove(std::get<Exception>(convertedScriptSource));
+    if (std::holds_alternative<Exception>(convertedScriptSource)) {
+        throwScope.clearException();
+    } else if (!std::holds_alternative<std::monostate>(convertedScriptSource)) {
```

Comment:
WebKit coding style omits braces for one-line if statement bodies:
```suggestion
    if (std::holds_alternative<Exception>(convertedScriptSource))
        throwScope.clearException();
    else if (!std::holds_alternative<std::monostate>(convertedScriptSource)) {
```